doc-changelog/parse_pr_title.py

```diff
@@ -321,7 +323,9 @@ def remove_existing_types(types, changelog_sections: list):
     for group in types:
         # Remove changelog section if it exists under [[tool.towncrier.type]] so that
         # only missing sections are appended to the pyproject.toml file
-        changelog_sections.remove(group["directory"])
+        section = group["directory"]
```

Comment:
```suggestion
        section = group.get("directory")
```
I'm guessing it's a dict. If for whatever reason dictionary is not defined, this will immediately crash. Better to use get. If you are sure "directory" is always going to exist, you can disregard this comment.